pages/message-sent/index.tsx

```diff
@@ -25,7 +27,7 @@ export default function MessageSent() {
                         <text className="font-inter font-normal">🕵️‍♀️ Get your own anonymous messages! </text>
                         <text className="font-inter font-normal">🔔 See what’s new in your school or community</text>
                     </div>
-                    <button className=" w-full bg-cyan hover:bg-cyan10 text-white font-lato flex py-2 px-4 rounded-lg justify-center items-center gap-2 text-justify">
+                    <button onClick={()=> window.location.href = publicRuntimeConfig.DOWNLOAD_BETTERSOCIAL_APP_URL} className=" w-full bg-cyan hover:bg-cyan10 text-white font-lato flex py-2 px-4 rounded-lg justify-center items-center gap-2 text-justify">
```

Comment:
any concern why not using next useRouter hook?

Reply:
actually no, will change to userouter hook